at_client/src/main/java/org/atsign/config/ConfigReader.java

```diff
@@ -13,13 +13,15 @@
  * Loads, reads and returns properties from the configuration file in the resources
  */
 public class ConfigReader {
-    private Map<String, String> config;
+    private Map<String, Map<String, String>> config;
+    private Map<String,String>propertyMap;
 
-    public String getProperty(String property) throws StreamReadException, DatabindException, FileNotFoundException{
+    public String getProperty(String property, String subProperty) throws StreamReadException, DatabindException, FileNotFoundException{
```

Comment:
Please preserve the getProperty(String property) for backwards compatibility and either add a new method, getNestedProperty(String property, String subProperty) or getNestedProperty(List<String> propertyNames) or similar